Some Fixes for SLS

Chaosyr opened the pull request:
* Fixed Set-Name/Code
* Added Token Cost and Flavor [Hopefully didn't butcher the setup for the cost…]

Chaosyr committed:
Make SLS have proper id of CTI

Custom TCG Inscryption is actually called what it should be called

Chaosyr committed:
Add Cost, Token, and Flavor to CTI

Also changed it to call the correct name

Chaosyr committed:
Merge branch 'Mouthless-Stoat:master' into master

Chaosyr committed:
Merge pull request #1 from SaxbyMod/patch-2

Add Cost, Token, and Flavor to CTI

Mouthless-Stoat reviewed (changes requested):
You should test code before putting PR to avoid wasting time and get rejected. Also blinding copying code (assume so cus there some pretty obvious mistake) is generally bad, you should try to understand it all before copying. Apart from the few small problem this PR could get accepted.

Mouthless-Stoat commented on extra/stoatLordCardStuff.js:
```diff
@@ -1,7 +1,7 @@
 
 var imfJson = {
-    ruleset: "Stoat Lord Card Thingymagik",
+    ruleset: "Custom TCG Inscryption",
```
Name should stay

Chaosyr replied:
Its not called that tho T_T

Mouthless-Stoat replied:
But it should stay nonetheless. Usually I don't change code name after being releases.

Mouthless-Stoat commented on extra/stoatLordCardStuff.js:
```diff
@@ -31,6 +31,33 @@ async function load() {
         cardFormated.health = card["Health"]
 
+        // parsing cost
+        card["Cost"] = card["Cost"]
```
Missing a `.toLowerCase()`. Without lowering them most of the case won;t be match. Aug doesn't require this because it is already lower, though I should add it

Chaosyr replied:
I see can fix that rq

Mouthless-Stoat commented on extra/stoatLordCardStuff.js:
```diff
@@ -31,6 +31,33 @@ async function load() {
+            } else if (["sapphire", "ruby", "emerald", "prism"].some((i) => cost.includes(i))) {
+                if (!cardFormated["gem"]) cardFormated["gem"] = []
+                for (let i = 0; i < temp[0]; i++) {
+                    cardFormated["gem"].push(temp[1])
```
It should be `mox` not `gem` refer to https://github.com/Mouthless-Stoat/MagpieTutor/blob/d5cbd451fdb30244dc6a26d16c90b3102d50858f/index.js#L277

Chaosyr replied:
But my end calls for gem not mox but Ig?

Mouthless-Stoat replied:
You have to parse it to `gem` so the formatting can work. The code doesn't understand the `gem` field

Mouthless-Stoat commented on extra/stoatLordCardStuff.js:
```diff
@@ -31,6 +31,33 @@ async function load() {
+                }
+            } else if (temp.length > 0) {
+                cardFormated[temp[1]] = parseInt(temp[0])
+            }
```
Missing handle for `Free` case, unless you want to omit cost when it is free.

Chaosyr replied:
Can fix

Mouthless-Stoat reviewed (changes requested):
~~idk how to use github apparently~~

Chaosyr committed:
Fix for mentioned things

Not including the name change because the proper name here is whats shown

Chaosyr committed:
Applied a thing to the wrong thing on accident

Chaosyr committed:
Syntax Fixing

Chaosyr committed:
Basic DMC and CTI Support

Desafts Mod (CTI)
Custom TCG Inscryption

*May want to remove the `.replaceAll("[TOKEN]", cardFormated.token)`*

Mouthless-Stoat commented on index.js:
```diff
@@ -526,13 +531,37 @@ const SetList = {
-    sls: {
-        name: "stuff",
+    cti: {
+        name: "custom tcg inscryption",
```
I still want to keep these name

Mouthless-Stoat commented on index.js:
```diff
@@ -526,13 +531,37 @@ const SetList = {
+            { name: "side", condition: 'card.tier == "Side Deck"' },
+            { name: "joke", condition: `card.tier == "Common (Joke Card)"` },
+            { name: "banned", condition: `card.temple == "Terrain/Extras"` }
+        ],
```
Don't intent on adding in a new set

Mouthless-Stoat commented on index.js:
```diff
@@ -291,7 +291,10 @@ const SetFormatList = {
-                info: [{ text: "**Token**: {token}", type: "sub" }],
+                info: [{ text: "**Token**: {token}", type: "sub" },
+                    { text: "\nThis card's format is: {format}", type: "sub"},
+                    { text: "\n\nThis card was provided by the Inscryption PvP wiki found [here.]({wikipage} '{name}')", type: "sub"},
```
This doesn't provide any info so it don't need to be there

Mouthless-Stoat commented on .gitattributes:
```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```
No

Mouthless-Stoat reviewed (changes requested):
Again, I do not intent to rename any of the set or add new one.

Chaosyr committed:
Delete .gitattributes

Chaosyr committed:
Fix pointed out issues

[Not changing them on my end for the record, this is just so `Stuff` as you call it is fixed on your end.

Mouthless-Stoat commented on index.js:
```diff
@@ -563,7 +562,7 @@ const SetList = {
         ],
     },
-
+*/
```
These can just be remove as again don;t intent on adding it

Chaosyr committed:
Update index.js

Switch the lines for Format and Wiki

Chaosyr committed:
Update DesaftsModCTIProcesser.js

Removes the `.replaceAll("[TOKEN]", cardFormated.token)` as it doesnt work properly.

Chaosyr committed:
Update CustomTCGInscryptionProcesser.js

Removes the `.replaceAll("[TOKEN]", cardFormated.token)` as it doesnt work properly.

Chaosyr committed:
Update index.js

Remove dmc set
